test/Chirp.Infrastructure.Tests/Repositories/CheepRepositoryTests.cs

```diff
@@ -31,6 +31,20 @@ public void TestAddCheep()
         Assert.Equal(cheep.Text, addedCheep.Text);
         Assert.True(addedCheep.Timestamp.ToFileTimeUtc() > DateTime.UtcNow.Add(TimeSpan.FromSeconds(-1)).ToFileTimeUtc());
     }
+
+    [Fact]
+    public void DeleteCheepTest()
+    {
+        //Arrange
+        CheepDto cheepDto = _mockCheepRepository.CheepRepository.GetCheepsForPage(1).First();
+        //Act
+        string cheepId = cheepDto.CheepId.ToString();
+        string cheepAuthor = cheepDto.AuthorName;
+        bool actualVal = _mockCheepRepository.CheepRepository.DeleteCheep(cheepId, cheepAuthor);
+        //Assert
+        Assert.True(actualVal);
```

Comment:
Looks good, but we should probably have an `Assert` to ensure that the `DbContext` `Delete` and `SaveChanges` functions are called. Look at lines 27 & 28 as inspiration.